src/util/url.js

```diff
@@ -1,9 +0,0 @@
-const createUrlQueryString = obj => {
-  const keys = Object.keys(obj).sort((a, b) => b < a)
-  return keys.map(k => `${k}=${obj[k]}`).join('&')
-}
-
-export default (url, p = {}) => {
```

Comment:
We were using this to build predictable keys for the localStorage cache. However, the `cache` object is pretty much tailor made for storing HTTP responses and assets so we just need to pass it the request object and it handles the key name